client/views/admin.html

```diff
@@ -8,6 +8,18 @@
 
       <div id="content" class="ui segment">
 
+        <h2>all active admin</h2>
+        <ul>
+          <li ng-repeat="(key, value) in admins">{{ value }} ({{ key }})</li>
+        </ul>
+
+        <h3>add admin</h3>
```

Comment:
i think this could be renamed to `modify admin` since you can also remove admins